Source/Particles/WarpXParticleContainer.H

```diff
@@ -1,12 +1,17 @@
 #ifndef WARPX_WarpXParticleContainer_H_
 #define WARPX_WarpXParticleContainer_H_
 
+#include <memory>
```

Comment:
Could you put this one down in the list of `include` (basically last)? Or is there a problem doing that?

Reply:
Sure! Placing this at the beginning of the ``` include ```  list was a mistake.